src/site/stages/build/drupal/metalsmith-drupal.js

```diff
@@ -105,29 +97,48 @@ function paginatePages(
       }
       for (let num = start; num < start + length; num++) {
         innerPages.push({
-          href: num === pageNum ? null : `${pagePath}${paginationPath(num)}`,
+          href:
+            num === pageNum
+              ? null
+              : `${page.entityUrl.path}${paginationPath(num)}`,
           label: num + 1,
           class: num === pageNum ? 'va-pagination-active' : '',
         });
       }
 
       pagedPage.paginator = {
         ariaLabel,
-        prev: pageNum > 0 ? `${pagePath}${paginationPath(pageNum - 1)}` : null,
+        prev:
+          pageNum > 0
+            ? `${page.entityUrl.path}${paginationPath(pageNum - 1)}`
+            : null,
         inner: innerPages,
         next:
           pageNum < pagedEntities.length - 1
-            ? `${pagePath}${paginationPath(pageNum + 1)}`
+            ? `${page.entityUrl.path}${paginationPath(pageNum + 1)}`
             : null,
       };
     }
 
     files[
-      `drupal${pagePath}${paginationPath(pageNum)}/index.html`
+      `drupal${page.entityUrl.path}${paginationPath(pageNum)}/index.html`
     ] = createFileObj(pagedPage, layout);
   }
 }
 
+// Return page object with path, breadcrump and title set.
+function createEntityUrl(page, drupalPagePath, title) {
+  const pathSuffix = title.replace(/\s+/g, '-').toLowerCase();
+  const generatedPage = Object.assign({}, page);
+  generatedPage.entityUrl.breadcrumb.push({
+    url: { path: drupalPagePath },
+    text: page.title,
+  });
+  generatedPage.entityUrl.path = `${drupalPagePath}/${pathSuffix}`;
+  generatedPage.title = title;
+  return generatedPage;
+}
+
```

Comment:
Is this only for press releases, or is it planned to be used for other pages as well?

Reply:
We're gonna use it on other pages as well. Anything that doesn't have a corresponding page entity in Drupal - the events listing page, the press release listing page, the services pages, etc.